addons/point_of_sale/models/account_bank_statement.py

```diff
@@ -15,3 +15,40 @@ class AccountBankStatementLine(models.Model):
     _inherit = 'account.bank.statement.line'
 
     pos_statement_id = fields.Many2one('pos.order', string="POS statement", ondelete='cascade')
+
+class AccountBankStmtCashWizard(models.Model):
```

Comment:
There is already an override of the model 'account.bank.statement.cashbox' in file pos_config.py, it is useless to make to override of the same class in the same module.

Move all content of this override in the one of pos_config.py